src/backend/vulkan/TextureVk.cpp

```diff
@@ -96,8 +96,14 @@ namespace backend { namespace vulkan {
                         VK_ACCESS_COLOR_ATTACHMENT_READ_BIT | VK_ACCESS_COLOR_ATTACHMENT_WRITE_BIT;
                 }
             }
-
-            // TODO(cwallez@chromium.org): What about present? Does it require VK_MEMORY_READ_BIT?
+            if (usage & nxt::TextureUsageBit::Present) {
+                // There is no access flag for present because the VK_KHR_SWAPCHAIN extension says
+                // that vkQueuePresentKHR makes the memory of the image visible to the presentation
+                // engine. There's also a not explicitly saying this is what dstAccessMask should be
```

Comment:
"not explicitly" -> "note explicitly"?
"this is what dstAccessMask should be 0" -> ?

Reply:
Done